packages/variable-editor/src/protocol/ClientContextProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import type { ReactNode } from 'react';
+import { createContext, useContext } from 'react';
+import type { Client, ClientContext } from './types';
+
+const defaultClientContext: any = undefined;
+const ClientContextInstance = createContext<ClientContext<any>>(defaultClientContext);
+
+export const useClient = (): Client<any> => {
```

Comment:
why do you need to use any here?

Reply:
Because this is a `.tsx` file, it seems to misinterpret type arguments:
![image](https://github.com/axonivy/config-editor-client/assets/141232142/16744b8a-4fa1-41d5-919f-f08f8667208c)
I could move this function to a different file though if you think that's better.

Reply:
add a comma: export const useClient = <T,>():